docs/en/engines/table-engines/special/memory.md

```diff
@@ -36,6 +36,8 @@ Upper and lower bounds can be specified to limit Memory engine table size, effec
   - Requires `max_rows_to_keep`
 - `max_rows_to_keep` — Maximum rows to keep within memory table where oldest rows are deleted on each insertion (i.e circular buffer). Max rows can exceed the stated limit if the oldest batch of rows to remove falls under the `min_rows_to_keep` limit when adding a large block.
   - Default value: `0`
+- `compress` - Whether to compress data in memory.
+  - Default value: `false`
```

Comment:
Why is it bool and not codec?

Reply:
what i can say, because it was implemented this way. prob it was seen as a bit of overkill and given that `Memory` is not a real storage but just an in-memory buffer maybe it really is.